### PR TITLE
[core]: fix spotless linter

### PR DESCRIPTION
**Pull request by Jaguar0625**
problem: spotless is not running
    solution: update spotless configuration
              add fixing of imports

### CODE REVIEW

File: core/src/it/java/org/nem/core/math/TunedMtjSparseMatrix.java [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
+import java.util.*;
 import no.uib.cipr.matrix.*;
 import no.uib.cipr.matrix.sparse.CompRowMatrix;
 
-import java.util.*;
-
-/**
- * A Matrix implementation that uses CompRowMatrix from MTJ.
- */
+/** A Matrix implementation that uses CompRowMatrix from MTJ. */
```
Comment (Wayonb) [inline core/src/it/java/org/nem/core/math/TunedMtjSparseMatrix.java:7]: Should we keep the docs as before?

```
/**
  * A Matrix implementation that uses CompRowMatrix from MTJ.
  */
  ```

File: nis/src/main/java/org/nem/nis/dao/retrievers/MosaicSupplyRetriever.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+import org.nem.core.model.primitive.*;
+import org.nem.nis.dao.*;
+import org.nem.nis.dbmodel.*;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Class for for retrieving mosaic supplies.
+ */
```
Comment (Wayonb) [inline nis/src/main/java/org/nem/nis/dao/retrievers/MosaicSupplyRetriever.java:16]: I guess spotless needs to be run on the new files?  so it should be a single line.

**Review by Wayonb — APPROVED** [review]
👍🏾

File: nis/src/main/java/org/nem/nis/dao/retrievers/MosaicSupplyRetriever.java (hunk 2 above)
Comment (Jaguar0625) [reply nis/src/main/java/org/nem/nis/dao/retrievers/MosaicSupplyRetriever.java:16]: this PR currently only has changes for core, but if rules look ok, i'll update it for the other projects as well

**Comment by Jaguar0625** [comment]
Close and replace with #576

**Pull request closed without merging**